core/build/src/lib.rs

```diff
@@ -176,6 +178,20 @@ impl SgxParseCallbacks {
         self
     }
 
+    /// Types to derive constant time for
+    ///
+    /// # Arguments
+    /// * `default_types` - Types to derive default for.
```

Comment:
:wrench: Wrong docs, though one could probably omit the arguments from the docs as it's named clearly and I think the other comments were me just being pedantic. It looks like rust documentation suggest not documenting arguments if they are clearly communicated by name.
```suggestion
    /// * `constant_time_types` - Types to derive constant time for.
```

Reply:
Good catch. Changed it